usb/dev.c

```diff
@@ -622,13 +733,18 @@ static void usb_devUnbind(usb_dev_t *dev)
 	int i;
 
 	for (i = 0; i < dev->nports; i++) {
-		if (dev->devs[i] != NULL)
+		if (dev->devs[i] != NULL) {
 			usb_devUnbind(dev->devs[i]);
+		}
 	}
 
+	usb_devFreeOids(dev);
```

Comment:
Shouldn't `usbdev_common.devOids` also be protected by `usbdev_common.lock`? Also includes the `usb_devOnDrvBindCb` callback.